src/platform/plugins/shared/home/public/application/components/solutions_section/solution_panel.tsx

```diff
@@ -54,3 +52,29 @@ export const SolutionPanel: FC<Props> = ({ addBasePath, solution }) => {
     </EuiFlexItem>
   );
 };
+
+const styles = ({ euiTheme }: UseEuiTheme) =>
+  css({
+    [`@media (min-width: ${euiTheme.breakpoint.m}px)`]: {
+      maxInlineSize: `calc(33.33% - ${euiTheme.size.m} * 10)`,
```

Comment:
@paulinashakirova why is 10 used here? 
This creates the problem for not occupying all the space for the items:

<img width="945" height="1111" alt="Image" src="https://github.com/user-attachments/assets/cc8987dc-0bd3-4bc7-9db7-ddbd0b812ea7" />